### ISSUE TITLE
OIDC broken after upgrading 6.0.1 to 6.3.2

### ISSUE THREAD

**Issue by ethernic** [IgnisDa/ryot#890]
I'm currently running Authentik (2024.6.0) and recently upgraded from ryot 6.0.1 to 6.3.2 and this broke SSO. The following error is all I see in the logs when trying to log in. When I get redirected back to `<FRONTEND_URL>/api/auth` I just get a "you must be logged in" notice in the UI.

```
[frontend] Error: No route matches URL "/favicon.ico"
[frontend]     at getInternalRouterError (/home/ryot/node_modules/@remix-run/router/router.ts:5353:5)
[frontend]     at Object.query (/home/ryot/node_modules/@remix-run/router/router.ts:3462:19)
[frontend]     at handleDocumentRequest (/home/ryot/node_modules/@remix-run/server-runtime/dist/server.js:268:35)
[frontend]     at requestHandler (/home/ryot/node_modules/@remix-run/server-runtime/dist/server.js:160:24)
[frontend]     at /home/ryot/node_modules/@remix-run/express/dist/server.js:41:28
```

The URL it's failing to match has changed once. The first time this happened it was a `.js` file but I don't recall the name and I can't get it to show up again. 

I have ryot deployed with podman quadlets using these unit and .env files:

```
# ryot.container
[Unit]
Requires=ryot-db.service

[Container]
ContainerName=ryot
EnvironmentFile=/opt/podman/ryot/.env
HostName=ryot
Image=ghcr.io/ignisda/ryot:v6.3.2
Label=backup.enable=true service.name=ryot compose.path=/opt/podman/ryot
Network=backend
Network=frontend
NoNewPrivileges=true
PublishPort=25352:8000

[Service]
Restart=on-abnormal

[Install]
WantedBy=default.target
```

```
# ryot-db.container
[Container]
ContainerName=ryot-db
EnvironmentFile=/opt/podman/ryot/.env
HostName=ryot-db
Image=docker.io/postgres:16.3-alpine
Label=backup.enable=true database.type=postgres service.name=ryot-db compose.path=/opt/podman/ryot
Network=backend
NoNewPrivileges=true
Volume=/opt/podman/ryot/data/db_data:/var/lib/postgresql/data:Z
Volume=/opt/podman/ryot/data/backups:/backups:Z

[Service]
Restart=on-abnormal

[Install]
WantedBy=default.target
```

```
# .env
PGID=1000
PUID=1000
TZ=America/Phoenix

POSTGRES_DB=ryot
POSTGRES_HOST=ryot-db
POSTGRES_PASSWORD=REDACTED
POSTGRES_USER=ryot

VIDEO_GAMES_TWITCH_CLIENT_ID=REDACTED
VIDEO_GAMES_TWITCH_CLIENT_SECRET=REDACTED

DATABASE_URL=postgres://ryot:REDACTED@ryot-db:5432/postgres


FRONTEND_URL=https://ryot.int.REDACTED.com
SERVER_OIDC_CLIENT_ID=REDACTED
SERVER_OIDC_CLIENT_SECRET=REDACTED
SERVER_OIDC_ISSUER_URL=https://id.REDACTED.com/application/o/ryot/
```

Authentik is also running in a quadlet and is functioning fine. The only other app I have hooked up to it is Immich which is not having any issues.

**Comment by niscolas** [comment]
Same error here, was running without issues until the update, also using Authentik

**Comment by niscolas** [comment]
Ok, workaround for now (at least to my problem) was to downgrade to v6.3.1

**Comment by ethernic** [comment]
> Ok, workaround for now (at least to my problem) was to downgrade to v6.3.1

This worked for me as well.

**Comment by IgnisDa** [comment]
Fixed with `v6.3.3`. Thanks for bringing it to my notice.
